geanyprj/src/geanyprj.c

```diff
@@ -74,6 +74,7 @@ static void reload_project(void)
 	{
 		if (g_current_project)
 			xproject_close(TRUE);
+		set_non_ft_build_wdir((gchar*)"");
```

Comment:
should take a `const gchar *` as it doesn't need modifying the string anyway, and would remove the need for this odd cast